wordpress-mobile/WordPress-Android
# Issue/9426 domain registration prompt

Fixes #9426

If the user is on a business plan, without a custom domain, and with unclaimed domain credit we prompt them to register a domain. 

This PR is a part of a bigger feature.

Test 1:
- Install wasabi build.
- Switch to a site on a business plan with unclaimed domain credit.
- Navigate to Plugins screen, and from there to details of any plugin (like Woocommerce).
- Tap on Install button.
- Observe the prompt to register a domain.

Test 1:
- Install wasabi build.
- Switch to a site on a business plan with "wasted" domain credit (use credit to register a domain and cancel it immediately).
- Navigate to Plugins screen, and from there to details of any plugin (like Woocommerce).
- Tap on Install button.
- Confirm that we are in normal Plugin installation flow (you will see "Install plugin" dialog)
